File: src/Equinox.Domain/Models/OrderBitfinex.cs
using System;
using Equinox.Domain.Core.Models;
namespace Equinox.Domain.Models
{
    public class OrderBitfinex : BaseBitfinex
    {
        public string symbol { get; set; }
        public string amount { get; set; }
        public string price { get; set; }
        public string side { get; set; }
        public string type { get; set; }
        public string exchange { get; set; }
        public bool is_hidden { get; set; }
        public bool is_postonly { get; set; }
        public bool ocoorder { get; set; }
        public string buy_price_oco { get; set; }
        public string sell_price_oco { get; set; }        

    }
}

File: src/Equinox.UI.Spa/ClientApp/app/components/bitfinex/model/OrderBitfinex.ts
export class OrderBitfinex {
    symbol: string;
    amount: string;
    price: string;
    side: string;
    type: string;
    is_hidden: boolean;
    is_postonly: boolean;
    ocoorder: boolean;
    buy_price_oco: number;
    sell_price_oco: number;
    private exchange: string;
    request: string;
    nonce: string;
    constructor() {
        this.exchange = 'bitfinex';
    }
}